sdk/eventgrid/eventgrid/tests.yml

```diff
@@ -1,8 +1,9 @@
 trigger: none
 
-extends:
-  template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
-  parameters:
-    PackageName: "@azure/eventgrid"
-    ResourceServiceDirectory: eventgrid
-    TestSamples: false
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/eventgrid"
+      ServiceDirectory: eventgrid
+      MatrixFilters:
+        - DependencyVersion: ^$
```

Comment:
I assume this is the new exclude min/max. It might be difficult for some folks to understand that but I think it is ok for now. (If only devops yaml supported c style macros :)... no...)

Reply:
Yep, agreed, it's non-obvious. I could add a yaml comment?